openslides/mediafiles/static/templates/mediafiles/mediafile-list.html

```diff
@@ -155,8 +155,8 @@ <h1 translate>Files</h1>
           </i>
       </tr>
     <tbody>
-      <tr ng-repeat="mediafile in mediafilesFiltered = (mediafiles | osFilter: filter.search : getFilterString |
-          filter: {filetype: filter.showPDFs} | orderBy: sortColumn:reverse)"
+      <tr ng-repeat="mediafile in mediafilesFiltered = (mediafiles | privateFilter | osFilter: filter.search : getFilterString |
+	  filter: {filetype: filter.showPDFs} | orderBy: sortColumn:reverse )"
```

Comment:
It appears I finally forgot to change Vim's whitespace settings before pushing :/